Makefile

```diff
@@ -1,5 +1,4 @@
 SEVERITIES = HIGH,CRITICAL
-SHELL := /bin/bash -x
 
 UNAME_M = $(shell uname -m)
 ARCH=
```

Comment:
should we take these out, since they're no longer used? And remove the `--build-arg ARCH` from below as well?

Reply:
sure I can take them out, the makefile is not really being used 100% except for the log target

Reply:
yeah lets just rip out all the ARCH stuff to avoid confusion

Reply:
actually the next iteration of modifications to our repo is revamping the Makefile so that it can work to a similar way to GHA using buildx